packages/redux/src/analytics/middlewares/setUser.ts

```diff
@@ -61,7 +72,7 @@ const getActionTypes = (actionTypes, defaultActionTypes) => {
 };
 
 // Reference to the current user data
-let currentUser;
+let currentUser: Record<string, unknown> | null;
```

Comment:
Same as the above ☝️

Reply:
Not applicable, this acts on the result of the user profile data and not the analytics user. thanks!

Reply:
Same as this one: https://github.com/Farfetch/blackout/pull/247/files#r841812391

You could add a type alias for the user data just for this file (However, that type should be kept in sync with the loadProfile type in useUserProfile hook, but let's not complicate here).